src/components/Card/Card.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { ReactNode, ReactElement } from "react";
+import styled, { css } from "styled-components";
+
+export interface CardProps {
+  children: ReactNode;
+  withHover?: boolean;
+}
+
+export const Card = ({ children, ...props }: CardProps): ReactElement => (
+  <StyledCard {...props}>{children}</StyledCard>
+);
+
+const StyledCard = styled.section<CardProps>`
+  background: ${({ theme }) => theme.colors.surface};
+  box-shadow: ${({ theme }) => `0px 0px 0px ${theme.colors.background}00`};
+  border-radius: 0.5rem;
```

Comment:
Will this translate to 8px? I'm used to just put the 8px, wondering how you've come up with 0.5 rem :)

Reply:
It's generally better to use `rem`, especially for compatibility with modern browser and IOS devices (`1rem = 16px`). Harder to get into it but produce good results

Reply:
Nice! I'll keep this in mind and start using `rem` going forward